core/src/components/tab-button/tab-button.md.scss

```
@@ -75,6 +75,26 @@
   height: $tab-button-md-badge-size-empty;
 }
 
+:host ::slotted(ion-badge[vertical="bottom"]) {
+  @include position($tab-button-md-badge-bottom-position, null, null, null);
+}
+
+:host ::slotted(ion-badge[vertical="bottom"]:empty) {
+  @include position($tab-button-md-badge-bottom-position-empty, null, null, null);
+}
```

Since we don't change any of the other value, we can depend on the defaults. I recommend reviewing how `position` works.

```suggestion
:host ::slotted(ion-badge[vertical="bottom"]) {
  @include position($tab-button-md-badge-bottom-position);
}

:host ::slotted(ion-badge[vertical="bottom"]:empty) {
  @include position($tab-button-md-badge-bottom-position-empty);
}
```